### ISSUE TITLE
Volume control doesn't catch processes from different inputs

### ISSUE THREAD

**Issue by Jeremy2131** [radj307/volume-control#159]
Hello, so I bought a new headset and the problem is I can route different apps through different inputs (Game/Media/Chat) but Volume control catches processes only from the Game input because Game is the default one
Can it be counted as a bug or a feature to be introduced?

**Comment by Dregu** [comment]
Yep seems like if you use the "App volume and device preferences" in W10 to set a non-default output for an app, VolumeControl will just adjust sessions in the default device for everything anyway. Except if you launch the other app first and then restart VolumeControl it will properly find the session on the other device and can even jump between sessions on different output devices, so clearly there's support for this, it's just bugged.

**Comment by teadrinker2015** [comment]
I'm not quite understand the circumstance you're describing. I'll elaborate mine anyway, to facilitate the replication of this issue hopefully.
- Foobar2000's output set to a speaker, Firefox output to the default device. At the beginning, default device is also that speaker.
- Then, I connect to a bluetooth headphone. consequently, it become the default now, and Firefox is redirect to the headphone, perfectly. Then I adjust volume.
- Everything is broken.

Sorry, I'm not intend to make a joke, but I can't remember the exact problem I had encountered. I recently replaced my bluetooth adapter to an audio-only bluetooth adapter, which act as an always-on audio output device, ignorant of which and whether my haedphone is connected. I just realized this change  solved my problem.
That's just the last piece of my whole workaround solutions package. Maybe I just want to share this to people.

If you're using a generic normal bluetooth adapter, buy another one. (as previously described)
